adagrams/game.py

```diff
@@ -1,11 +1,91 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+LETTER_VALUES = {
+    "A" : 1, "E" : 1, "I" : 1, "O" : 1, "U" : 1,
+    "L" : 1, "N" : 1, "R" : 1, "S" : 1, "T" : 1, 
+    "D" : 2, "G" : 2,
+    "B" : 3, "C" : 3, "M" : 3, "P" : 3,
+    "F" : 4, "H" : 4, "V" : 4, "W" : 4, "Y" : 4,
+    "K" : 5,
+    "J" : 8, "X" : 8,
+    "Q" : 10,"Z" : 10
+    }
+
+LONG_WORD_BONUS = 8
+LONG_WORD_RANGE = (7, 8, 9, 10)
+
+DRAW_POOL = [key for key, value in LETTER_POOL.items() for value in range(value)]
```

Comment:
Awesome work in using constants in the preceding lines and this _stunning_ list comprehension!